aws/ecs/service-no-alb-with-volume/README.md

```diff
@@ -0,0 +1,50 @@
+# aws/ecs/service-no-alb - EC2 Container Service Service/Task without load balancer
+This module is used to create an ECS service as well as task definition
+
+## What this does
+
+ - Create task definition
+ - Create service
+
+## Required Inputs
+
+ - `cluster_id` - ID for ECS Cluster
+ - `service_name` - Name of service, all lowercase, no spaces.
+ - `service_env` - Name of environment, used in naming task definition. Ex: `staging`
+ - `container_def_json` - JSON for container definition.
+ - `desired_count` - Number of tasks to run in service
+ - `volume_name` - Name for volume
+ - `volume_host_path` - Path on host EC2 instance to mount volume
+
+### Optional Inputs
+
+ - `task_role_arn` - ARN for role to assign to task definition. Default: `blank`
+ - `network_mode` - Networking mode for task. Default: `bridge`
+ - `deployment_maximum_percent` - Upper limit of tasks that can run during a deployment. Default: `200`%
+ - `deployment_minimum_healthy_percent` - Lower limit of tasks that must be running during a deployment. Default: `50`%
+
+## Outputs
+
+ - `task_def_arn` - ARN for task definition.
+ - `task_def_family` - Family name of task definition.
+ - `task_def_revision` - Revision number of task definition.
```

Comment:
You're missing "task_def_revision_via_data"

Reply:
added